strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryStreamSupport.java

```diff
@@ -74,11 +81,21 @@ private void open()
         
         logger.debug("Locking [{}].", path);
         
-        Lock lock = ctx instanceof RepositoryStreamWriteContext ? lockSource.writeLock() : lockSource.readLock();
+        //TODO: ArtifactManagementServiceImplTest.testConcurrentReadWrite
+        //Lock lock = ctx instanceof RepositoryStreamWriteContext ? lockSource.writeLock() : lockSource.readLock();
```

Comment:
@sbespalov : What is this `//TODO` for? Is this dead code? If not, could you please add a follow-up task, if this is indeed necessary? Thanks! :)